src/abstractdataframe/io.jl

```diff
@@ -163,25 +85,30 @@ function html_escape(cell::AbstractString)
 end
 
 function Base.show(io::IO, ::MIME"text/html", df::AbstractDataFrame)
-    n = size(df, 1)
     cnames = _names(df)
     write(io, "<table class=\"data-frame\">")
-    write(io, "<thead>")
+    write(io, "<thread>")
```

Comment:
Can you put this in a separate commit? We should merge it in DataTables first, and then backport it to here (along with other changes merged since I prepared this branch).